src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -487,7 +488,8 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     offset: number | null = null,
     reqConfig: RequestConfiguration,
     findTilesAdditionalParameters: FindTilesAdditionalParameters,
-  ): Promise<PaginatedTiles> {
+    distinct: string | null = null,
```

Comment:
Is there are reason why this is a distinct (sic) parameter and not a part of `findTilesAdditionalParameters`?

Reply:
No, not really. Was changed.